Build.PL

```diff
@@ -289,6 +289,7 @@ my $requires = {
                 'Digest::SHA1'                    => '0.0',
                 'English'                         => '1.02',
                 'MIME::Base64'                    => 0,
+                'perl'                            => '5.14',
```

Comment:
Thanks.



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.